### ISSUE TITLE
Port to 1.18

### ISSUE THREAD

**Issue by enderprism** [CaffeineMC/hydrogen-fabric#65]

**Comment by zimdin12** [comment]
It would be nice if it would be also compatible with 1.18.1

**Comment by JoeMan97** [comment]
Any update about the hydrogen 1.18.1 release??

**Comment by Pectacular** [comment]
![image](https://user-images.githubusercontent.com/54170867/147911477-d6271432-aa42-412b-bf0e-494761a58fe5.png)

**Comment by yum13241** [comment]
Hydrogen 1.18.1 would be VERY useful.

**Comment by TenthCrusadering** [comment]
Check Actions, there's a build that you can download for 1.18.1!

**Comment by yum13241** [comment]
thanks! now to hope it works...

**Comment by lrutjens** [comment]
> Check Actions, there's a build that you can download for 1.18.1!

Does it work on servers?

**Comment by Pectacular** [comment]
Since they have disabled Github Actions, here's the build -
[hydrogen-fabric-mc1.18.1-0.3-SNAPSHOT.zip](https://github.com/CaffeineMC/hydrogen-fabric/files/7902050/hydrogen-fabric-mc1.18.1-0.3-SNAPSHOT.zip)

**Comment by lrutjens** [comment]
> Since they have disabled Github Actions, here's the build -
> [hydrogen-fabric-mc1.18.1-0.3-SNAPSHOT.zip](https://github.com/CaffeineMC/hydrogen-fabric/files/7902050/hydrogen-fabric-mc1.18.1-0.3-SNAPSHOT.zip)

Does this work on servers?

**Comment by Pectacular** [comment]
> > Since they have disabled Github Actions, here's the build -
> > [hydrogen-fabric-mc1.18.1-0.3-SNAPSHOT.zip](https://github.com/CaffeineMC/hydrogen-fabric/files/7902050/hydrogen-fabric-mc1.18.1-0.3-SNAPSHOT.zip)
> 
> Does this work on servers?

Yes, but Hydrogen isn't very beneficial on servers. If you want to reduce your memory usage on servers more effectively, then use [FerriteCore](https://www.curseforge.com/minecraft/mc-mods/ferritecore-fabric)
